### ISSUE TITLE
[Fixed in GH-6052] No visual indicator when focus is on a checkbox

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#3777]
**Environment Details**
Spartacus: develop
Browser: Chrome
Device: Mac
Happens on: https://dev-com-2.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD/

**Steps to Reproduce**
- In SPA, Add product to cart.
- Proceed to Checkout
- On Shipping Add Form, press the key "Tab" to move from field to field.

**Actual Result**
- When the focus is on the "Set as default", there's no visual indicator. 
- For the text fields, the fields become grey. 
- But for the checkbox, nothing... 

<img width="919" alt="Screen Shot 2019-07-25 at 2 32 31 PM" src="https://user-images.githubusercontent.com/45409515/61899436-711f4b00-aee9-11e9-8bc8-f2bb089de698.png">

**Comment by KateChuen** [comment]
still an issue.

**Comment by developpeurweb** [comment]
Seem OK now for all 3 browsers.

**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 4 07 16 PM](https://user-images.githubusercontent.com/15113219/73574543-f9a41880-4443-11ea-94c9-3c7e9dbcce98.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
